hazelcast-client/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -759,4 +760,47 @@ public ClientConfig setUserCodeDeploymentConfig(ClientUserCodeDeploymentConfig u
         this.userCodeDeploymentConfig = userCodeDeploymentConfig;
         return this;
     }
+
+    /**
+     *
+     * @param mapName The name of the map for which the query cache config is to be returned.
+     * @param cacheName The name of the query cache config.
+     * @return The query cache config. If the config does not exist, it is created.
+     */
+    public QueryCacheConfig getOrCreateQueryCacheConfig(String mapName, String cacheName) {
+        Map<String, Map<String, QueryCacheConfig>> allQueryCacheConfig = getQueryCacheConfigs();
```

Comment:
Yes, this is continous query caches. it can be defined multiple on a map. The first is the map name and the second is the query cache name.